level1/f256/modules/vtio.asm

```diff
@@ -397,9 +441,18 @@ Term                ldx       >D.OrgAlt   get the original alternate IRQ vector
                     pshs      u
                     ldu       V.KeyDrvMPtr,u
                     os9       F$Unlink
+                    puls      u
+                    ifgt      Level-1
```

Comment:
Why is this conditional here if we support Level 1?

Reply:
Removed support for Level-1 to remove from FEU. Since FEU uses L1 vtio, it was easier to strip the mouse from L1 than to make a conditional for the FEU.